src/transformers/models/blip_2/convert_blip_2_original_to_pytorch.py

```diff
@@ -79,6 +82,12 @@ def create_rename_keys(config):
     # QFormer
     rename_keys.append(("Qformer.bert.embeddings.LayerNorm.weight", "qformer.layernorm.weight"))
     rename_keys.append(("Qformer.bert.embeddings.LayerNorm.bias", "qformer.layernorm.bias"))
+    rename_keys.append(("Qformer.bert.embeddings.word_embeddings.weight", "embeddings.word_embeddings.weight"))
```

Comment:
I just found that i got an error converting `blip2-opt-2.7b` model
`KeyError: 'Qformer.bert.embeddings.word_embeddings.weight'`

i will need to do this rename keys for the _itm_ models only

Reply:
Yes it'd be great to keep backwards compatibility for the existing checkpoints, and also making sure that users don't get warnings when loading the existing checkpoints (like `unexpected keys in checkpoint` etc.)

Reply:
- I fixed the keys issues on the convert script
- then the logit comparison on convert script for `blip2-opt-2.7b` were failing, I needed to revert back a change that i did, now the scale is done after the dot product between "query" and "key"
- slow tests on test_modeling_blip_2.py are passing. `RUN_SLOW=1 python -m pytest tests/models/blip_2/test_modeling_blip_2.py`
- but the tests on CI are failing, it look for a reason different than this PR

The convert script for `blip2-opt-2.7b` is failing in the generation step, it looks like it is not related to the changes of this PR, because i got the same error on the main branch.
```ValueError: Input length of input_ids is 0, but `max_length` is set to -14. This can lead to unexpected behavior. You should consider increasing `max_length` or, better yet, setting `max_new_tokens`.```

Reply:
cc @zucchini-nlp who fixed this error, we might need to update the conversion script of BLIP-2 for that

Reply:
Can you check that model's generation config has a high enough `max_length` (if it doesn't have `max_length` the default is 20)?

Right now BLIP will count all tokens (image and text) towards `max_length`, so we can either add higher `max_length` or `max_new_tokens` in model's generation config so that one run generate directly with `model.generate(**inputs)`

Reply:
I added `max_length=50` to the `generate` call of the lavis model on the convert script, now conversion of `blip2-opt-2.7b` works